src/stim/stabilizers/tableau_pybind_test.py

```diff
@@ -57,6 +57,20 @@ def test_from_named_gate():
         stim.Tableau.from_named_gate("X_ERROR")
 
 
+def test_from_state_vector():
```

Comment:
It seems even if I remove the random scale the following does not hold:

```
t = stim.Tableau.random(n)
t2 = stim.Tableau.from_state_vector(t.to_state_vector(), endian='little')
assert t == t2 # <-- fails
```

`assert stim.Tableau.from_conjugated_generators(\n    xs=[\n        stim.PauliString("+Y"),\n    ],\n    zs=[\n        stim.PauliString("+X"),\n    ],\n) == stim.Tableau.from_conjugated_generators(\n    xs=[\n        stim.PauliString("+Z"),\n    ],\n    zs=[\n        stim.PauliString("+X"),\n    ],\n)`

However if I instead compare `t.to_state_vector() == t2.to_state_vector()` then they are equal, so I could use that instead or try to figure out why t1and t2 aren't equal.

Reply:
This, however, does work:

```
t = stim.Tableau.random(n)
t2 = stim.Tableau.from_state_vector(t.to_state_vector(endian='little'), endian='little')
np.testing.assert_array_equal(t.to_stabilizers(canonicalize=True), t2.to_stabilizers(canonicalize=True))
```
So it seems `==` is a little tricky because it doesn't canonicalize. Let me know if the above would work.

Reply:
yeah that's fine to canonicalize it